examples/example_omb_csv_reader.py

```diff
@@ -0,0 +1,51 @@
+import logging
```

Comment:
examples need to have a title: https://github.com/OpenDrift/trajan/blob/main/examples/example_drifters.py#L1 otherwise the documentation build is going to fail.